src/vivarium_public_health/disease/model.py

```diff
@@ -132,8 +137,6 @@ def on_initialize_simulants(self, pop_data: SimulantData) -> None:
 
         assert self.initial_state in {s.state_id for s in self.states}
 
-        # FIXME: this is a hack to figure out whether or not we're at the simulation start based on the fact that the
-        #  fertility components create this user data
```

Comment:
Is this just no longer a hack?

Reply:
I don't know if it's a hack or not, but it's been around for 5+ years. It doesn't seem terrible and clearly it's not a priority to do anything about it, in any case.